packages/x-charts/src/ScatterChart/ScatterMarker.tsx

```diff
@@ -0,0 +1,72 @@
+import * as React from 'react';
+import PropTypes from 'prop-types';
+import { SeriesId } from '../models/seriesType/common';
+import { useScatterSeries } from '../hooks';
+
+export interface ScatterMarkerProps {
+  ownerState: ScatterMarkerOwnerState;
+  /**
+   * Callback fired when clicking on a scatter item.
+   * @param {MouseEvent} event Mouse event recorded on the `<svg/>` element.
+   */
+  onClick?: (event: React.MouseEvent<SVGElement, MouseEvent>) => void;
+}
+
+export interface ScatterMarkerOwnerState {
```

Comment:
We can remove this and merge the props into the `ScatterMarkerProps`

Reply:
This is used in `Scatter.types.ts` and on `ScatterMarkerProps`, so it's probably better to keep it to avoid duplication.

I'll remove it if we remove one usage.